src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -181,10 +179,44 @@ def get_device_metadata(self) -> dict:
 
         return device
 
+    def run_conversion(
+        self,
+        nwbfile_path: Optional[FilePathType] = None,
+        nwbfile: Optional[NWBFile] = None,
+        metadata: Optional[dict] = None,
+        overwrite: bool = False,
+        stub_test: bool = False,
+        starting_time: Optional[float] = None,
+        write_as: Optional[str] = None,
+        write_electrical_series: bool = True,
+        es_key: str = None,
+        compression: Optional[str] = None,
+        compression_opts: Optional[int] = None,
+        iterator_type: str = "v2",
+        iterator_opts: Optional[dict] = None,
+    ):
+        super().run_conversion(
+            nwbfile_path=nwbfile_path,
+            nwbfile=nwbfile,
+            metadata=metadata,
+            overwrite=overwrite,
+            stub_test=stub_test,
+            starting_time=starting_time,
+            write_as=write_as or "raw",
+            write_electrical_series=write_electrical_series,
+            es_key=es_key or "ElectricalSeriesRaw",
+            compression=compression,
+            compression_opts=compression_opts,
+            iterator_type=iterator_type,
+            iterator_opts=iterator_opts,
+        )
+
```

Comment:
If you were to just adjust the default args to "raw" and "ElectricalSeriesRaw" the behavior would be more clear from the function signature. Is there an advantage to doing it this way?

Reply:
> Is there an advantage to doing it this way?

Not really, happy to change the default.

It should also be changed at a more general level anyway (may require a little more debugging for other interfaces, though) since this is a problem for any such interface

Reply:
Well, half generalized anyway. The LFP Interface can be adjusted that way at the top level, but the recording triggers some failures so I'll do it in a separate PR